eva-accession-release-automation/run_release_in_embassy/release_common_utils.py

```diff
@@ -55,7 +55,12 @@ def close_mongo_port_to_tempmongo(port_forwarding_process_id):
 def get_bgzip_tabix_commands_for_file(bgzip_path, tabix_path, file):
     commands = ["rm -f {0}.gz".format(file), "({0} < {1} > {1}.gz)".format(bgzip_path, file),
                 "({0} -f {1}.gz)".format(tabix_path, file)]
+    return commands
+
 
+def get_bgzip_bcftools_index_commands_for_file(bgzip_path, bcftools_path, file):
+    commands = ["rm -f {0}.gz".format(file), "({0} -c {1} > {1}.gz)".format(bgzip_path, file),
```

Comment:
Suggest
```suggestion
    commands = ["rm -f {0}.gz".format(file), "({0} --csi {1} > {1}.gz)".format(bgzip_path, file),
```
for clarity.